src/context/Baser.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Uri, window, workspace } from 'vscode'
+import { Store } from '.'
+import '../shared/extension'
+
+export class Baser {
```

Comment:
>Baser [](start = 13, length = 5)

Can we rename this to at least "PathBaser" and add a description as to what this code is for?